src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -70,6 +70,9 @@ let uniqueId = 0;
   selector: 'sky-autocomplete',
   templateUrl: './autocomplete.component.html',
   styleUrls: ['./autocomplete.component.scss'],
+  providers: [
+    SkyAutocompleteAdapterService
```

Comment:
Now that the adapter is provided by the component, we no longer need to generate a renderer using the `Renderer2Factory`.
Instead of this:
```
private renderer: Renderer2;

constructor(
  private rendererFactory: RendererFactory2
) {
  this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
}
```

We can now do this:
```
constructor(
  private renderer: Renderer2
) {}
```